src/developer-testing/developer-testing.controller.ts

```diff
@@ -27,4 +32,29 @@ export class DeveloperTestingController {
     }
     res.status(200).send({});
   }
+
+  @ApiOperation({
+    summary:
+      'Given a set of addresses this generates CDD claims, provides with a large amount of POLYX and makes them CDD providers',
+    description:
+      'This endpoint initializes a set of addresses to be chain admin accounts. The signer must best a CDD provider and have sufficient POLYX to cover the initial amounts (DEV ONLY)',
+  })
+  @Post('/create-admins')
+  async createAdmins(@Body() params: CreateCddProviders): Promise<IdentityModel[]> {
+    const ids = await this.developerTestingService.batchCreateAdmins(params);
+
+    return Promise.all(ids.map(id => createIdentityModel(id)));
+  }
```

Comment:
@ApiResponse is missing here.

Reply:
Also, just to be consistent I would recommend changing the return type to `Promise<ResultsModel<IdentityModel>>`. Can leave as it as well